### ISSUE TITLE
 table text cut off. 

### ISSUE THREAD

**Issue by MohammedAlJaff** [arXiv/html_feedback#433]
### Description

Text inside the first column of tables are chopped off at the start. 

### (Required) What is this issue most closely related to? Select one.

Formula

### Internal issue ID

32dd454d-1d26-4d7b-8ca3-19cfa2c6b730

### Paper URL

https://arxiv.org/html/2402.00351v1

### Browser

Chrome/121.0.0.0

### Device Type

Desktop

**Comment by dginev** [comment]
Thank you for the report. Here is a fragment link directly to the table in question:
https://arxiv.org/html/2402.00351v1#S4.T1

Diagnostic comment:

There is some experimental CSS from ar5iv which misfires here, and should likely be removed/updated, also taking in mind the new markup emitted from latexml v0.8.8.

I was fishing for a caption-friendly approach to inline tables (realized via `<span>`s) with `display:flex`, but that seems to allow the table to expand beyond its parent width. Some more investigation would be needed.

https://github.com/dginev/ar5iv-css/blob/7761e77eb556a5df83d65c1849e291790cac729b/css/ar5iv.css#L1442-L1443

**Comment by dginev** [comment]
Thank you for your report - the specific issue is now solved.

We have regenerated the article and you should be able to see an improvement.
